src/lib/rng/auto_rng/auto_rng.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+* (C) 2016 Jack Lloyd
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/auto_rng.h>
+#include <botan/entropy_src.h>
+#include <botan/loadstor.h>
+#include <botan/internal/os_utils.h>
+
+#if defined(BOTAN_HAS_HMAC_DRBG)
+  #include <botan/hmac_drbg.h>
+#endif
+
+#if defined(BOTAN_HAS_HMAC_RNG)
+  #include <botan/hmac_rng.h>
+#endif
+
+#if defined(BOTAN_HAS_SYSTEM_RNG)
+  #include <botan/system_rng.h>
+#endif
+
+namespace Botan {
+
+AutoSeeded_RNG::AutoSeeded_RNG(size_t max_output_before_reseed)
+   {
+   m_rng.reset(new BOTAN_AUTO_RNG_DRBG(MessageAuthenticationCode::create(BOTAN_AUTO_RNG_HMAC),
+#if defined(BOTAN_HAS_SYSTEM_RNG)
+                                       system_rng(),
+#else
+                                       Entropy_Sources::global_sources(),
+#endif
+                                       max_output_before_reseed));
+
+   if(!m_rng->is_seeded())
+      {
+      m_rng->next_byte();
+      BOTAN_ASSERT(m_rng->is_seeded(), "ok");
+      //throw Exception("AutoSeeded_RNG failed to generate seed material");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this block of commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=cpp%3ACommentedCode)